app/services/sign_in/errors.rb

```diff
@@ -5,9 +5,9 @@ module Errors
     class StandardError < StandardError
       attr_reader :code
 
-      def initialize(message:, code: Constants::ErrorCode::INVALID_REQUEST)
```

Comment:
Were you able to find where in the code it's calling the keyword argument as a hash? I'd rather fix this instead where this is being called, rather than here (I'd say it's not intended that this would be called with a hash instead of directly by params individually)

Reply:
(If you're able to find out where it's complaining exactly I can see what pattern is causing this warning and go ahead and fix all of the instances I can find Identity-related code)

Reply:
Hi @bosawt I'll remove this from the PR and create one just for the warning, then look into it further. I'll send you the link once I do. Thanks for looking and commenting!

Reply:
Could you remove `lib/saml/user_attributes/ssoe.rb` changes as well?

Reply:
BTW, please lemme know what you find out. I can get someone on my team to make the changes when necessary. I'm thinking we followed some pattern that unintentionally uses implicit hash as params and we probably just copy-pasta it over various places